convert2rhel/backup.py

```diff
@@ -189,33 +189,57 @@ def pop(self):
 
         return restorable
 
-    def pop_all(self):
+    def pop_all(self, _honor_partitions=False):
         """
         Restores all RestorableChanges known to the Controller and then returns them.
 
         :returns: List of RestorableChange objects that were known to the Controller.
         :raises IndexError: If there are no RestorableChanges currently known to the Controller.
 
         After running, the Controller object will not know about any RestorableChanges.
-        """
-        # This code ignores partitions.  Only restore_to_partition() honors
-        # them.
-        restorables = [r for r in self._restorables if r is not self.partition]
 
-        if not restorables:
+        .. note:: _honor_partitions is part of a hack for 1.4 to let us split restoring changes into
```

Comment:
Should this be 1.6?

Reply:
No, @abadger refers to https://github.com/oamg/convert2rhel/blob/main/convert2rhel/backup.py#L229 introduced in 1.4.